native/src/widget/slider.rs

```diff
@@ -407,13 +407,13 @@ pub fn draw<T, R>(
                 x: bounds.x,
                 y: line_y,
                 width: line_offset,
-                height: style.rail.rail_size,
+                height: style.rail.size,
             },
-            border_radius: 0.0.into(),
-            border_width: 0.0,
-            border_color: Color::TRANSPARENT,
+            border_radius: style.rail.border_radius.into(),
```

Comment:
do you want to switch this up so that is doesn't effect the gap between the two rail. so  rail.border_radius should be a 
`[f32; 4]` then the left rail would be  something like `[rail.border_radius[0], rail.border_radius[1], 0.0, 0.0]` and same below. using [2] and [3]. 

also idk what the order is for border_radius for corners so you would have to look which ones to use for each side

Reply:
I made it simple and kept it as a `f32` and handled the border_radius manually:

```rust
border_radius: [
    style.rail.border_radius,
    0.0,
    0.0,
    style.rail.border_radius,
]
```

I am not sure, design wise, it will ever wanted to put a radius on the "inner" corners of the rails so I don't what the user to think about it as well.